training/distributed_training/pytorch/model_parallel/gpt2/submit-train-gpt-simple.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>Clarify what CI means/stands for, for readers who are unfamiliar.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3034/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>fixed</p>